addons/cargo/CfgEventHandlers.hpp

```diff
@@ -88,4 +88,14 @@ class Extended_InitPost_EventHandlers {
             init = QUOTE(_this call DFUNC(initObject));
         };
     };
+    class RoadCone_F {
+        class ADDON {
+            init = QUOTE(_this call DFUNC(initObject));
+        };
+    };
+    class RoadCone_L_F {
```

Comment:
<code>RoadCone_F</code> and <code>RoadCone_L_F</code> is not part of <code>ReammoBox_F</code> its a part of <code>thingX</code>.